convergence/do_all_the_convergence_tests/compile_and_run.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import errno
+import re
+from argparse import ArgumentParser
+from shutil import copy
+from pathlib import Path
+import math
+
+### Settings for SuperMUC-NG
+# on_a_cluster = True
+# host_arch = 'skx'
+# orders = range(2,8)
+# resolutions = range(2,7)
+# compilers = ['mpiicc', 'mpiicpc', 'mpiifort']
+
+### Settings for Sebastians workstation
+on_a_cluster = False
+host_arch = "hsw"
+orders = range(3, 7)
+resolutions = range(2, 5)
+compilers = ["mpicc", "mpicxx", "mpif90"]
```

Comment:
Wouldn't it make sense to add these to the argument parser (with suitable defaults)?